common/azure_auth.go

```diff
@@ -185,7 +185,7 @@ func (aa *DatabricksClient) ensureWorkspaceURL(ctx context.Context,
 	}
 	resourceID := aa.AzureResourceID
 	if resourceID == "" {
-		return fmt.Errorf("somehow resource id is not set")
+		return fmt.Errorf("please set `azure_workspace_resource_id` provider argument")
```

Comment:
Should we clarify that similar to host clarification?

Reply:
This is when host is set and no resource id

Reply:
I meant more detailed explanation, like this: https://github.com/databrickslabs/terraform-provider-databricks/blob/master/common/client.go#L278